src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -1865,6 +1865,39 @@ def test_handling_intrinsics(code, expected_type, expected_op, symbol_table):
     if expected_type is not CodeBlock:
         assert assign.rhs._operator == expected_op, \
             "Fails when parsing '" + code + "'"
+        if isinstance(assign.rhs, UnaryOperation):
+            assert assign.rhs._named_args is None
+        else:
+            assert len(assign.rhs.children) == len(assign.rhs._named_args)
+            for named_arg in assign.rhs._named_args:
+                assert named_arg is None
+
+
+@pytest.mark.parametrize(
+    "code, expected_type, expected_op, expected_names",
+     [('x = sum(a, dim=1)', BinaryOperation, BinaryOperation.Operator.SUM, [None, "dim"]),
+      ('x = sum(a, mask=.true.)', BinaryOperation, BinaryOperation.Operator.SUM, [None, "mask"]),
+      ('x = sum(a, dim=1, mask=.true.)', NaryOperation, NaryOperation.Operator.SUM, [None, "dim", "mask"]),
+      ('x = sum(a, mask=.true., dim=1)', NaryOperation, NaryOperation.Operator.SUM, [None, "mask", "dim"]),
+      ('x = sum(a, 1, mask=.true.)', NaryOperation, NaryOperation.Operator.SUM, [None, None, "mask"])])
+@pytest.mark.usefixtures("f2008_parser")
+def test_handling_intrinsics_named_args(code, expected_type, expected_op, expected_names, symbol_table):
```

Comment:
A few of these lines are more than 80 chars, can you break them please.

Reply:
Oops.